### DISCUSSION TITLE
1.8.2

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#994] [Обновления]
## Список изменений

- Добавлена возможность выбрать несколько языков для параметра "Не переводить с родного языка" (#818 в #975)
- Добавлена возможность перемещения кнопки и субтитров на мобильных устройствах (в #978, #980)
- Добавлена поддержка Artstation (#58)
- Добавлена поддержка ленты для Twitter (X) (частично #981)
- Добавлена поддержка ленты для Douyin (#876)
- Исправлен баг, из-за которого при выборе субтитров не обновлялся заголовок и не происходило выделение в меню (в #975)
- Исправлен баг, из-за которого не активировался аудиоплеер со стримами
- Исправлено определение длительности видео на `/client/...` в Яндекс Диске (#973)
- Изменена логика активации проксирования субтитров (в #975)
- Убрано появление кнопки загрузки аудио файла при просмотре перевода стримов (в прямом эфире)

## What's Changed
* Не переводить с родного языка" для 2-х и более языков #818 by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/974
* Доработка субтитров и исправление бага в createVOTSelect by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/975
* Поддержка тачскрина by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/978
* Исправление #978 by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/980
* Fix #976 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/982
* Update 1.8.2 by @ilyhalight in https://github.com/ilyhalight/voice-over-translation/pull/993

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.8.1...1.8.2

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.8.2'>1.8.2</a>.</em>

**Comment by V3mp812** [comment]
спасибо за обновление, и за поддержку столь полезной вещи <3

**Comment by V1adBo** [comment]
После обновления с 1.8.1 на 1.8.2 в Firefox 128.5.2esr пропала кнопка расширения.
Удаление и переустановка не помогла, как и включение «Обход Media CSP».
Заработало только после возврата к 1.8.1 (хорошо, что у меня ещё Chrome есть, скопировал оттуда).
Где старые версии тут — не нашёл.

**Reply by ilyhalight** [reply]
В 1.8.3 исправлено. Старые версии во вкладке releases на главной репозитория
